UnityProject/Assets/Scripts/Objects/Doors/Modules/EmagInteractionModule.cs

```diff
@@ -39,15 +39,15 @@ public override ModuleSignal ClosedInteraction(HandApply interaction, HashSet<Do
 						foreach (var item in ItemStorage.GetNamedItemSlots(NamedSlot.id))
 						{
 							var ID = item.ItemAttributes;
+							Emag emagInIdSlot = item.Item?.OrNull().gameObject.GetComponent<Emag>()?.OrNull();
 
-							if (ID != null)
+							if (ID != null) continue;
+							if (emagInIdSlot != null) continue;
```

Comment:
```suggestion
							if (ID == null) continue;
							if (emagInIdSlot == null) continue;
```

Just noticed that

Reply:
why would we continue if it's null? do you want to throw NREs?

Reply:
You continue to avoid the NRE

Reply:
also `ID` isnt used anywhere so can be removed

Reply:
done